diff --git a/tests/test_basic.py b/tests/test_basic.py
index dfb63e3..1de4046 100644
--- a/tests/test_basic.py
+++ b/tests/test_basic.py
@@ -193,17 +193,6 @@ class TestBasic(unittest.TestCase):
         self.assertTrue(df.equals(self.df_usecols))
         self.assertTrue(meta.number_columns == len(self.usecols))
         self.assertTrue(meta.column_names == self.usecols)
-        
-    def test_sas7bdat_international(self):
-        """
-        On windows, paths with international characters are problematic. This is verifying that it is working as expected
-        """
-        # in addition, this works only in python 3
-        if sys.version_info[0]>2:
-            df, meta = pyreadstat.read_sas7bdat(os.path.join(self.international_data_folder, "sample.sas7bdat"))
-            self.assertTrue(df.equals(self.df_pandas))
-            self.assertTrue(meta.number_columns == len(self.df_pandas.columns))
-            self.assertTrue(meta.number_rows == len(self.df_pandas))
 
     def test_sas7bdat_nodates(self):
         df, meta = pyreadstat.read_sas7bdat(os.path.join(self.basic_data_folder, "sample.sas7bdat"), disable_datetime_conversion=True)
